spec/systems/view_keyword_list_spec.rb

```diff
@@ -3,6 +3,8 @@
 require 'rails_helper'
 
 describe 'views keyword list', type: :system do
+  include ActiveJob::TestHelper
```

Comment:
do we need this

Reply:
Good catch and I just realized that I overlooked the easiest way which I can just fabricate the keyword status.
Fixed in b044f736bd92e04013551ff632fcc57c6f6e1f30